index.js

```diff
@@ -1,244 +0,0 @@
-import { querySelect, querySelectAll, createElem } from "./utils.js";
-
-const cvc = querySelectAll(".security-number");
-const cardNum = querySelectAll(".card-number");
-const cardName = querySelectAll(".card-name");
-const expMonth = querySelectAll(".exp-month");
-const expYear = querySelectAll(".exp-year");
-
-const nameInput = querySelect("#user-name");
-const cardNumInput = querySelect("#user-card-number");
-const expMonthInput = querySelect("#exp-date-month");
-const expYearInput = querySelect("#exp-date-year");
-const cvcInput = querySelect("#cvc");
-
-const submitFormBtn = querySelect(".submit-form");
-const form = querySelect("form");
-const thankYouState = querySelect(".thank-you-state");
-const continueBtn = querySelect("#continue-btn");
-
-//Changes the informations on the card.
-const cardInfoChange = (input, changingElem) => {
-  changingElem.forEach((elem) => {
-    elem.innerText = input.value.toUpperCase();
-  });
-};
-
-//Limits the characters in the input.
-const charLimiter = (input, quantity) => {
-  if (input.value.length >= quantity) {
-    input.value = input.value.slice(0, quantity);
-  }
-};
-
-//Limits date inputs so the user cannot provide more than requested number in input field.
-const dateLimit = (input, quantity) => {
-  if (input.value > quantity) {
-    input.value = "";
-  }
-};
-
-//Provide the input data to the card.
-const cardNumChange = () => {
-  cardNum.forEach((elem) => {
-    if (elem.innerText === "0000 0000 0000 0000") {
-      elem.innerText = "";
-    }
-
-    elem.innerText = ` ${cardNumInput.value.slice(
```

Comment:
**question:** Why is there white space between ` and $, also at the end?

Reply:
I think I tried to make it look somehow better in VSCode but failed and it just was there.